From fe52922aa62e0c773915121214e768a0c160c95c Mon Sep 17 00:00:00 2001
From: Ryan Davis <ryand@zenspider.com>
Date: Thu, 14 Nov 2013 17:24:28 -0800
Subject: [PATCH] - Apparently getting files in a different order on osx on
 ruby 2.1. (sanemat) [git-p4: depot-paths = "//src/hoe/dev/": change = 9031]

(cherry picked from commit 8113b6a7cc579933816d78be1c04a32e40c840af)
---
 test/test_hoe.rb | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/test/test_hoe.rb b/test/test_hoe.rb
index ec87b54..21e8b70 100644
--- a/test/test_hoe.rb
+++ b/test/test_hoe.rb
@@ -173,7 +173,7 @@ class TestHoe < MiniTest::Unit::TestCase
     assert_equal ['email'], spec.email
     assert_equal ['sow'], spec.executables
     assert_equal text_files, spec.extra_rdoc_files
-    assert_equal files, spec.files
+    assert_equal files.sort, spec.files.sort
     assert_equal "https://github.com/seattlerb/hoe", spec.homepage
     # TODO: assert_equal "https://github.com/seattlerb/hoe", spec.metadata
     assert_equal ['--main', 'README.txt'], spec.rdoc_options
--
2.4.3
